girderformindlogger/api/v1/user.py

```diff
@@ -786,6 +786,12 @@ def getUsersDetails(self):
             default="",
             required=False
         )
+        .param(
+            'login',
+            'login username, Deprecated. Do not use.',
+            deprecated=True,
+            required=False
+        )
```

Comment:
Why add a new, immediately deprecated parameter to a deprecated API call rather than creating a new call? See https://github.com/childmindinstitute/mindlogger-app-backend/issues/243#issuecomment-574370443